runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc

```diff
@@ -1041,8 +1041,11 @@ OperationFactory::OperationFactory()
     OperandIndexSequence inputs{init_param.inputs[0]};
 
     operation::ResizeBilinear::Param param;
-    param.height_out = operands.at(OperandIndex{init_param.inputs[1]}).asScalar<int32_t>();
-    param.width_out = operands.at(OperandIndex{init_param.inputs[2]}).asScalar<int32_t>();
+    const auto size_op = operands.at(OperandIndex{init_param.inputs[1]}).asVector<int32_t>();
+    param.height_out = size_op[0];
+    param.width_out = size_op[1];
+    param.align_corners = operands.at(OperandIndex{init_param.inputs[2]}).asScalar<int32_t>();
+    param.half_pixel_centers = operands.at(OperandIndex{init_param.inputs[3]}).asScalar<int32_t>();
```

Comment:
Don't change NNAPI spec. We don't maintain NNAPI specification (NNAPI's onwer is google)
NNAPI specification don't define `align_corners` and `half_pixel_centers` for `ANEURALNETWORKS_RESIZE_BILINEAR` operatoin.

Reply:
Thanks for the explanation. I have since then made the changes.